meinberlin/apps/bplan/templates/meinberlin_bplan/emails/submitter_confirmation.de.email

```diff
@@ -1,11 +1,14 @@
 {% extends 'email_base.'|add:part_type %}
+
 {% load wagtailcore_tags %}
 
 {% block subject %}Ihre Stellungnahme zum Bebauungsplan "{{ project.name }}" von {{ module.active_phase.start_date|date:'d.m.Y' }} - {{ module.active_phase.end_date|date:'d.m.Y' }}{% endblock %}
```

Comment:
Shouldnt this be vom instead of von?

Reply:
Oh! I didn't read this, because it wasn't in the story... Will do it!

Reply:
Well, actually I will remove the dates as we also did that in the headline.

Reply:
Thats makes sense, yes!